config/crd/bases/k8s.raptor.ml_dataconnectors.yaml

```diff
@@ -1,10 +1,9 @@
----
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.9.2
-  creationTimestamp: null
+  creationTimestamp:
```

Comment:
This file is autogenerated, applying the auto-formatter here will force you to reapply everytime you run `make manifests`